waspc/src/Wasp/Util.hs

```diff
@@ -190,3 +192,9 @@ hexFromString = Hex
 
 hexToString :: Hex -> String
 hexToString (Hex s) = s
+
+allValues :: (Bounded a, Enum a) => [a]
+allValues = [minBound ..]
+
+quoteSingle :: String -> String
+quoteSingle s = "'" ++ s ++ "'"
```

Comment:
I tried to find whether something like this already exists in one of our dependencies, but couldn't find it.